nodes/celestia-node-store-structure.md

```diff
@@ -0,0 +1,28 @@
+---
+description: This section contains information on the celestia-node datastore and its contents.
+---
+
+# celestia-node datastore
+
+The node's datastore refers to the storage structure
+used to manage the data that supports the node's operation.
+It consists of directories and files that contain the node's state,
+configuration, and other information relevant to the node.
+
+The following are the directories and files found in the datastore:
+
+- `/blocks`: This directory stores blocks. Each file contained in this directory
+represents a block on Celestia and contains its associated data. However, the light node's datastore does not contain this directory, as light nodes do not store blocks.
```

Comment:
- [ ] Add clarification that it is present on bridge and full nodes, but not light nodes